packages/run-utils/package.json

```diff
@@ -13,9 +13,9 @@
   "scripts": {
     "prebuild": "rm -rf lib",
     "build": "tsc",
-    "test": "ava",
-    "test:dev": "ava -w",
-    "test:ci": "c8 -r lcovonly -r text -r json ava"
+    "test": "vitest run",
+    "test:dev": "vitest",
+    "test:ci": "vitest run --reporter=default"
```

Comment:
What is the diff between test and test:dev?

Reply:
`test:dev` is a watch mode and `test` is just normal execution without watch mode (we have that in every lib)

Reply:
cool